rotkehlchen/chain/manager.py

```diff
@@ -1248,3 +1248,25 @@ def get_eth2_history_events(
                 ))
 
         return defi_events
+
+    @cache_response_timewise()
+    def get_loopring_balances(self) -> Dict[Asset, Balance]:
+        """Query loopring balances if the module is activated"""
+        # Check if the loopring module is activated
+        loopring_module = self.get_module('loopring')
+        if loopring_module:
+            addresses = self.queried_addresses_for_module('loopring')
+            balances = loopring_module.get_balances(addresses=addresses)
+
+            # Now that we have balances for the addresses we need to agregate the
+            # assets in the different addresses
+            aggregated_balances: Dict[Asset, Balance] = {}
```

Comment:
turn this into a `defaultdict` just like we for example do here: 
https://github.com/rotki/rotki/blob/ff9138a796287e8eac7dfb3fa2b03700c657fb51/rotkehlchen/exchanges/kraken.py#L519

This way below.

```python
                    if asset in aggregated_balances:
                        aggregated_balances[asset] += balance
                    else:
                        aggregated_balances[asset] = balance

```

turns to

```python
aggregated_balances[asset] += balance
```

and probably some list or dict comprehension can make it more pythonic.

Reply:
I thought about doing this but was looking for what was the default value. I didn't notice the ZERO initialization.